_std/defines/delqueue.dm

```diff
@@ -0,0 +1,4 @@
+#define DELQUEUE_SIZE 35
```

Comment:
combine this and `lagcheck.dm` into something like `lag.dm` imo

Reply:
done, and did a bit more reordering in setup. there was some junk in lagcheck.dm that i missed!